src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -277,6 +277,34 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
       }
     }
 
+    case _: PNew => noMessages
+
+    case m@PMake(typ, args) =>
+      args.flatMap { arg =>
+        exprType(arg) match {
+          case IntT(config.typeBounds.Int) | IntT(config.typeBounds.UntypedConst) =>
```

Comment:
I would hav expected a check with `assignableTo.errors(exprType(e), ...)(n)`. Also, maybe a stupid question: does the length and capacity argument have to be an int or untyped constant? What about long? or unsigned int, etc?

Reply:
Yes, it has to be an `int` or, more precisely, according to the go spec
>  Each of the size arguments n and m must be of integer type or an untyped constant. A constant size argument must be non-negative and representable by a value of type int; if it is an untyped constant it is given type int.